datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1886,6 +1886,21 @@ type ParentContainersResult {
     containers: [Container!]!
 }
 
+"""
+Additional properties about EntityName
+"""
+type EntityNameProperties {
```

Comment:
so i see you using this as a means to add typing to your frontend properties. however we should only be adding things to the graphql schema that we plan to map back to the client, otherwise it would get confusing what's just existing as a type for the FE vs what exists as a returnable type from GMS

we can always define an `interface` or `type` in typescript if we need to cast objects on the frontend into a specific type and I would suggest we do that here instead

Reply:
As per the [last comment](https://github.com/datahub-project/datahub/pull/11585#discussion_r1815344337), now we are passing `parentDataset` object, so we don't require type to be defined for `properties`.